### DISCUSSION TITLE
mcap release artifact rust/v0.7.1 is now http 404

### DISCUSSION THREAD

**Discussion by paulm-planted** [foxglove/mcap#1121] [General] [answered]
Hi,  

I had added this in my Dockerfile:

```
# Install mcap and foxglove command line tools
RUN wget https://github.com/foxglove/mcap/releases/latest/download/mcap-linux-amd64 -O mcap
```
 
The latest mcap release artifact was available a week ago.
wget https://github.com/foxglove/mcap/releases/download/releases/rust/v0.7.1/mcap-linux-amd64

But its gone http 404 this week.  

What should I do?  

Thanks,
Paul

**Comment by defunctzombie** [comment, marked as answer]
The `mcap` repo is a mono-repo with different kinds of releases. If you want to use the CLI you need to link to a CLI specific release and get the assets from that release: https://github.com/foxglove/mcap/releases

**Reply by paulm-planted** [reply]
```
wget https://github.com/foxglove/mcap/releases/download/releases%2Fmcap-cli%2Fv0.0.32/mcap-linux-amd64 -O mcap
```
correct?

Just want to make sure, since this document has the link for the latest  (which results in 404). 
https://mcap.dev/guides/cli

**Reply by jtbandes** [reply]
Thanks for pointing that out, we will update the documentation soon.
